src/libs/Navigation/getTopMostCentralPaneRouteName.js

```diff
@@ -0,0 +1,32 @@
+import lodashFindLast from 'lodash/findLast';
+
+/**
+ * Find the name of top most central pane route.
```

Comment:
What is a "Central pane"? Is that the main right side content?

Reply:
Yes. It is a navigator responsible for the main right side content https://github.com/Expensify/App/blob/main/src/libs/Navigation/AppNavigator/Navigators/CentralPaneNavigator.js

Reply:
Personally I think that `getTopMostContentPaneRouteName` is a better name then, so that we stick to the same terminology as the doc used. Thoughts @hayata-suenaga ?

Reply:
Is the name "content pane" used somewhere in the docs?

Reply:
I was referring to this terminology. "central pane" was confusing to me since it's really the right side pane. I think referring to it as the content makes more sense. We can wait for Hayata's opinion though :) 


<img width="444" alt="2023-10-04_07-50-08" src="https://github.com/Expensify/App/assets/42391420/72250c4c-10bf-48be-b6f5-23bc7761e577">

Reply:
@stitesExpensify good point! Let's use `ContentPane`